larq/layers.py

```diff
@@ -56,6 +56,14 @@ def build(self, input_shape):
                     values_shape=self.kernel.shape, name=f"{self.name}/flip_ratio"
                 )
 
+    @property
+    def non_trainable_weights(self):
```

Comment:
We really need to find a better way to share logic between layers. I'll do some refactoring next week. We should also checkout how TensorFlow is handling their automatic mixed precision [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/mixed_precision/experimental/autocast_variable.py) to maybe gain some inspiration.